bin/plugin/commands/changelog.js

```diff
@@ -47,20 +47,51 @@ const manifest = require( '../../../package.json' );
  */
 
 /**
- * Mapping of label names to grouping heading text to be used in release notes,
- * intended to be more readable in the context of release notes. Also used in
- * merging multiple related groupings to a single heading.
+ * Mapping of label names to sections in the release notes.
+ *
+ * Labels are sorted by the priority they have when there are
+ * multiple candidates. For example, if an issue has the labels
+ * "[Block] Navigation" and "[Type] Bug", it'll be assigned the
+ * section declared by "[Block] Navigation".
  *
  * @type {Record<string,string>}
  */
 const LABEL_TYPE_MAPPING = {
-	Bug: 'Bug Fixes',
-	Regression: 'Bug Fixes',
-	Feature: 'Features',
-	Enhancement: 'Enhancements',
-	'New API': 'New APIs',
-	Experimental: 'Experiments',
-	Task: 'Various',
+	'[Block] Navigation': 'Experiments',
+	'[Block] Query': 'Experiments',
+	'[Block] Post Comments Count': 'Experiments',
+	'[Block] Post Comments Form': 'Experiments',
+	'[Block] Post Comments': 'Experiments',
+	'[Block] Post Featured Image': 'Experiments',
+	'[Block] Post Hierarchical Terms': 'Experiments',
+	'[Block] Post Title': 'Experiments',
+	'[Block] Site Logo': 'Experiments',
```

Comment:
Just tested the PR and it works great. as a follow-up we might want to group more things automatically for instance, create a sublist for all FSE related block PRs 

``
### Experiments:

 - Full Site Editing Blocks:
    - entry 1
    - entry 2
```

There are more things that could use this "sub grouping".